mappings/net/minecraft/util/dynamic/Codecs.mapping

```diff
@@ -20,7 +24,36 @@ CLASS net/minecraft/class_5699 net/minecraft/util/dynamic/Codecs
 			COMMENT the first codec
 		ARG 1 second
 			COMMENT the second codec
+	METHOD method_36240 createNonEmptyListChecker ()Ljava/util/function/Function;
+	METHOD method_36241 rangedInt (IILjava/util/function/Function;)Lcom/mojang/serialization/Codec;
+		ARG 0 min
+		ARG 1 max
+		ARG 2 messageFactory
+	METHOD method_36242 (Ljava/lang/Integer;)Ljava/lang/String;
+		ARG 0 v
+	METHOD method_36243 createRangeChecker (Ljava/lang/Number;Ljava/lang/Number;Ljava/util/function/Function;)Ljava/util/function/Function;
+		ARG 0 min
+		ARG 1 max
+		ARG 2 messageFactory
+	METHOD method_36244 (Ljava/lang/Number;Ljava/lang/Number;Ljava/util/function/Function;Ljava/lang/Number;)Lcom/mojang/serialization/DataResult;
+		ARG 3 value
+	METHOD method_36245 (Ljava/util/List;)Lcom/mojang/serialization/DataResult;
+		ARG 0 list
+	METHOD method_36246 (Ljava/lang/Integer;)Ljava/lang/String;
+		ARG 0 v
+	METHOD method_36973 nonEmptyList (Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
+		ARG 0 originalCodec
+	METHOD method_37349 (Ljava/util/List;)Lcom/mojang/serialization/DataResult;
+		ARG 0 suppliers
+	METHOD method_37350 (Ljava/util/function/Supplier;)Lcom/mojang/serialization/DataResult;
+		ARG 0 supplier
+	METHOD method_37351 createPresentValuesChecker ()Ljava/util/function/Function;
+	METHOD method_37352 createPresentValueChecker ()Ljava/util/function/Function;
 	CLASS class_5839 Xor
+		COMMENT An xor codec that only permits exactly one of the two data choices to be
```

Comment:
```suggestion
		COMMENT A xor codec that only permits exactly one of the two data choices to be
```

Reply:
it's pronounced "eks-oh-are", or "exclusive or", both of which are suitable with "an" than "a" for they start with a vowel sound by pronunciation.